src/main/resources/db/migration/data/R__9_12__OFFENCE_INDICATORS.sql

```diff
@@ -1 +1,2 @@
 INSERT INTO OFFENCE_INDICATORS (OFFENCE_INDICATOR_ID, OFFENCE_CODE, STATUTE_CODE, INDICATOR_CODE) VALUES (-1, 'RC86355', 'RC86', 'S15/CJIB');
+INSERT INTO OFFENCE_INDICATORS (OFFENCE_INDICATOR_ID, OFFENCE_CODE, STATUTE_CODE, INDICATOR_CODE) VALUES (-2, 'RC86355', 'RC86', 'SOR');
```

Comment:
Adding another indicator to the test data here made the test fail with the old query approach.